File: aoj/volume3/n0315/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int programmingScore = sc.nextInt();
        int mobileScore = sc.nextInt();
        int cgScore = sc.nextInt();

        int totalScore = programmingScore + mobileScore + cgScore;

        System.out.println(totalScore);

    }

}

File: aoj/volume3/n0315/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int programmingScore;
    int mobileScore;
    int cgScore;
    cin >> programmingScore >> mobileScore >> cgScore;

    int totalScore = programmingScore + mobileScore + cgScore;

    cout << totalScore << endl;

    return 0;

}
